docs/docsite/rst/community/maintainers_guidelines.rst

```diff
@@ -119,14 +123,82 @@ Some other general guidelines to encourage contributors:
+* [ ] If the pull request is not a documentation fix, it must include a :ref:`changelog fragment <collection_changelog_fragments>`. Check the format carefully as follows:
+
+  * New modules and plugins (that are not jinja2 filter and test plugins) do not need changelog fragments.
+  * For jinja2 filter and test plugins, check out the `special syntax for changelog fragments <https://github.com/ansible-community/antsibull-changelog/blob/main/docs/changelogs.rst#adding-new-roles-playbooks-test-and-filter-plugins>`_.
+  * The changelog content contains useful information for end users of the collection.
+* [ ] If new files are added with the pull request, they follow the `licensing rules <https://github.com/ansible-collections/overview/blob/main/collection_requirements.rst#licensing>`_.
+* [ ] The changes follow the :ref:`Ansible documentation standards <developing_modules_documenting>` and the :ref:`style_guide`.
+* [ ] The changes follow the :ref:`Development conventions <developing_modules_best_practices>`.
+* [ ] If a new plugin is added, it is one of the `allowed plugin types <https://github.com/ansible-collections/overview/blob/main/collection_requirements.rst#modules-plugins>`_.
+* [ ] Documentation, examples, and return sections use FQCNs for the ``M(..)`` :ref:`format macros <module_documents_linking>` when referring to modules.
+* [ ] Modules and plugins from ansible-core use ``ansible.builtin.`` as an FQCN prefix when mentioned.
+* [ ] When a new option, module, plugin, or return value is added, the corresponding documentation or return sections use ``version_added:`` containing the *collection* version which they will be first released in.
+
+  * This  is typically the next minor release, sometimes the next major release. For example: if 2.7.5 is the current release, the next minor release will be 2.8.0, and the next major release will be 3.0.0).
+* [ ] FQCNs are used for ``extends_documentation_fragment:``, unless the author is referring to doc_fragments from ansible-core.
+* [ ] New features have corresponding examples in the :ref:`examples_block`.
+* [ ] Return values are documented in the :ref:`return_block`.
+
+
+Review tests in the PR
+----------------------
+Review the following if tests are applicable and possible to implement for the changes included in the PR:
+
+
+* [ ] Where applicable, the pull request has :ref:`testing_integration` and :ref:`testing_units`.
+* [ ] All changes are covered. For example, a bug case or a new option separately and in sensible combinations with other options.
+* [ ] Integration tests cover ``check_mode`` if supported.
+* [ ] Integration tests check an actual state of the system, not only what the module reports. For example, if the module changes a file, check that the file was actually changed by using the ``ansible.builtin.stat`` module..
```

Comment:
```suggestion
* [ ] Integration tests check the actual state of the system, not only what the module reports. For example, if the module changes a file, check that the file was changed by using the ``ansible.builtin.stat`` module..
```

Reply:
i would leave `actually` here as it can look like the file changed but actually didn't, i.e. the word emphasizes this a bit.